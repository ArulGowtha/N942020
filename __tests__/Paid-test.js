import React from 'react';

import renderer from 'react-test-renderer';
import Paid from '../src/screens/Paid';

test('renders correctly', () => {
  const tree = renderer.create(<Paid />).toJSON();
  expect(tree).toMatchSnapshot();
});
