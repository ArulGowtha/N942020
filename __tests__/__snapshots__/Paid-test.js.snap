// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "lightgrey",
      "flex": 10,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1.5,
        "justifyContent": "center",
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "lightgrey",
          "borderRadius": 7,
          "flexDirection": "row",
          "height": 50,
          "justifyContent": "space-between",
          "marginHorizontal": 15,
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        placeholder="Search here"
        rejectResponderTermination={true}
        style={
          Object {
            "fontSize": 18,
            "width": "80%",
          }
        }
        underlineColorAndroid="transparent"
      />
      <Text
        allowFontScaling={false}
        style={
          Array [
            Object {
              "color": undefined,
              "fontSize": 12,
            },
            Object {
              "alignSelf": "center",
              "fontSize": 40,
              "marginHorizontal": 15,
            },
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
            Object {},
          ]
        }
      >
        
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "white",
        "flex": 8.5,
      }
    }
  >
    <View
      style={
        Object {
          "backgroundColor": "lightgrey",
          "flex": 1,
        }
      }
    >
      <RCTScrollView
        collapsable={false}
        data={
          Array [
            Object {
              "name": "Jeeva Raj",
            },
            Object {
              "name": "Ananth Palani",
            },
            Object {
              "name": "venkat Veera",
            },
            Object {
              "name": "Rahul Jag",
            },
            Object {
              "name": "Muruga John",
            },
            Object {
              "name": "Durai Govind",
            },
            Object {
              "name": "Jagan Manikam",
            },
            Object {
              "name": "Gokul George",
            },
            Object {
              "name": "Sabrish Dinesh",
            },
            Object {
              "name": "kumar Bharath",
            },
            Object {
              "name": "Vinoth Rajan",
            },
            Object {
              "name": "Venkat Rajan",
            },
          ]
        }
        disableVirtualization={false}
        getItem={[Function]}
        getItemCount={[Function]}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        onContentSizeChange={[Function]}
        onEndReachedThreshold={2}
        onGestureHandlerEvent={[Function]}
        onGestureHandlerStateChange={[Function]}
        onLayout={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        renderScrollComponent={[Function]}
        scrollEventThrottle={50}
        stickyHeaderIndices={Array []}
        updateCellsBatchingPeriod={50}
        viewabilityConfigCallbackPairs={Array []}
        windowSize={21}
      >
        <View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Jeeva Raj
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Ananth Palani
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  venkat Veera
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Rahul Jag
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Muruga John
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Durai Govind
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Jagan Manikam
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Gokul George
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  Sabrish Dinesh
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            onLayout={[Function]}
            style={null}
          >
            <View
              style={
                Object {
                  "flex": 1,
                }
              }
            >
              <View
                accessible={true}
                focusable={false}
                onClick={[Function]}
                onResponderGrant={[Function]}
                onResponderMove={[Function]}
                onResponderRelease={[Function]}
                onResponderTerminate={[Function]}
                onResponderTerminationRequest={[Function]}
                onStartShouldSetResponder={[Function]}
                style={
                  Object {
                    "backgroundColor": "white",
                    "flexDirection": "row",
                    "height": "80%",
                    "justifyContent": "space-between",
                    "marginBottom": "10%",
                    "marginHorizontal": 18,
                    "opacity": 1,
                  }
                }
              >
                <Text
                  style={
                    Object {
                      "alignSelf": "center",
                      "fontSize": 20,
                      "marginLeft": "2%",
                      "width": "40%",
                    }
                  }
                >
                  kumar Bharath
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "orange",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Entypo",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
                <Text
                  allowFontScaling={false}
                  style={
                    Array [
                      Object {
                        "color": "green",
                        "fontSize": 12,
                      },
                      Object {
                        "alignSelf": "center",
                        "fontSize": 30,
                        "marginRight": "5%",
                      },
                      Object {
                        "fontFamily": "Ionicons",
                        "fontStyle": "normal",
                        "fontWeight": "normal",
                      },
                      Object {},
                    ]
                  }
                >
                  
                </Text>
              </View>
            </View>
          </View>
          <View
            style={
              Object {
                "height": 0,
              }
            }
          />
        </View>
      </RCTScrollView>
    </View>
  </View>
</View>
`;
